src/config-schema.coffee

```diff
@@ -155,32 +172,36 @@ module.exports =
       softTabs:
         type: 'boolean'
         default: true
+        description: 'If the `Tab Type` config setting is set to "auto" and autodetection of tab type from buffer content fails, then this config setting determines whether a soft tab or a hard tab will be inserted when the Tab key is pressed.'
```

Comment:
> This is really only for a new file. Auto will autodetect, and fall back to this for new files

Thanks for mentioning this, @benogle -- I think that config settings related to soft tabs are among the ones causing most confusion so I'd like the descriptions to be :gem: clear.

Can you clarify what _exactly_ "only for a new file" means? Does this mean new files created in Atom, files opened in Atom for the first time (I'm _guessing_ it's this), or something else? In other words, in which situations is the description above not true?